src/django_components/middleware.py

```diff
@@ -1,2 +1,4 @@
 # These middlewares are part of public API
-from django_components.dependencies import ComponentDependencyMiddleware as ComponentDependencyMiddleware  # noqa: F401
+from django_components.dependencies import ComponentDependencyMiddleware
+
+__all__ = ["ComponentDependencyMiddleware"]
```

Comment:
And same for the middlewares. All middlewares defined in this file will be picked up by the docs-generating script.